xmlChart.go

```diff
@@ -544,6 +545,9 @@ type formatChartAxis struct {
 		Italic    bool   `json:"italic"`
 		Underline bool   `json:"underline"`
 	} `json:"num_font"`
+	Scaling struct {
+		LogBase float64 `json:"logbase"`
```

Comment:
I suggest `LogBase` as a field of the struct `formatChartAxis`, instead of `Scaling`’s

Reply:
I used `Scaling` because from what I saw in the `charts*.xml` file, the `LogBase` option was put under `Scaling`:
```xml
```